it-play-30/src/test/scala/uk/gov/hmrc/support/HtmlComparisonTest.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import org.jsoup.Jsoup
+import org.jsoup.helper.W3CDom
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import javax.xml.transform.TransformerFactory
+import javax.xml.transform.dom.DOMSource
+import javax.xml.transform.stream.StreamResult
+import java.io.StringWriter
+
+class HtmlComparisonTest extends AnyFlatSpec with Matchers {
+
+  def normalizeHtml(html: String): String = {
```

Comment:
`normalize` or `normalise`...?

Reply:
@kyle-bowden That's weird, the functiona definition here is still reading as "normalize" for me?